src/core/lib/event_engine/posix_engine/ev_epoll1_linux.h

```diff
@@ -46,7 +47,8 @@ namespace experimental {
 class Epoll1EventHandle;
 
 // Definition of epoll1 based poller.
-class Epoll1Poller : public PosixEventPoller {
+class Epoll1Poller : public PosixEventPoller,
+                     public grpc_event_engine::experimental::Forkable {
```

Comment:
This can just be `public Forkable` since we're already in that namespace.

Reply:
Ah yes. Didn't make the connection.